tensorflow/tools/ci_build/ci_test_on_install.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
That should be in the pip.sh. And it already is there ;-)
echo "Unrecognized container type: ${CONTAINER_TYPE}"
exit 1